docs/website/docs/getting-started/quickstart.md

```diff
@@ -1,163 +1,226 @@
+Pick `nodejs` to create the frontend component:
+```shell
+cd frontend
+odo create nodejs frontend
+```
 
+Create a URL in order to access the component in the browser:
 ```shell
-cd backend
-odo create java-springboot backend
-odo url create --port 8080 --host `minikube ip`.nip.io
-odo push
+odo url create --port 3000 --host <CLUSTER-HOSTNAME>
 ```
 
-The `minikube ip` command helps get the IP address of the minikube instance. It is required to create a URL accesible from the web browser of the host system on which minikube is running.
+**Minikube users:** Use `minikube ip` to find out the hostname and then use `<MINIKUBE-HOSTNAME>.nip.io`  for `--host`.
 
-## Create the Postgres database
 
-In the [prerequisites](#prerequisites) section, we installed Postgres Operator. Before being able to create a service using it, first ensure that the Operator is installed correctly. You should see the Postgres Operator like in below output. Note that you might see more Operators in the output if there are other Operators installed on your cluster: 
+Push the component to the cluster:
 ```shell
-odo catalog list services
+odo push
 ```
 
+The URL will be listed in the `odo push` output, or can be found in `odo url list`.
+
+Browse the site and try it out! Note that you will not be able to add, remove or list the to-dos yet, as we have not linked the frontend and the backend components yet.
+
+## Create the backend Java component
+
+The backend application is a Java Spring Boot based REST API which will list, insert and delete to-dos from the database.
+
+Find `java-springboot` in the catalog:
 ```shell
-$ odo catalog list services
-Services available through Operators
-NAME                        CRDs
-postgresoperator.v5.0.3     PostgresCluster
+odo catalog list components
 ```
 
-If you don't see the Postgres Operator here, it might be still installing. Take a look at what you see in the `PHASE` column in below output:
+Example output of `odo catalog list components`:
 ```shell
-kubectl get csv
+Odo Devfile Components:
+NAME                             DESCRIPTION                                                         REGISTRY
+java-quarkus                     Quarkus with Java                                                   DefaultDevfileRegistry
+java-springboot                  Spring Boot® using Java                                             DefaultDevfileRegistry
+java-vertx                       Upstream Vert.x using Java                                          DefaultDevfileRegistry
+...
 ```
 
+Let's create the component below:
 ```shell
-$ kubectl get csv                         
-NAME                      DISPLAY                           VERSION   REPLACES                  PHASE
-postgresoperator.v5.0.3   Crunchy Postgres for Kubernetes   5.0.3     postgresoperator.v5.0.2   Succeeded
+cd ../backend
+odo create java-springboot backend
+odo url create --port 8080 --host <CLUSTER-HOSTNAME>.nip.io
```

Comment:
I don't think you should be writing `<CLUSTER-HOSTNAME>.nip.io` here. Instead, a note for minikube users like you have in nodejs component above makes more sense.